test/inductor/test_torchinductor.py

```diff
@@ -7116,6 +7116,33 @@ def forward():
             compiled = compile_fx_inner(mod, ())
             assert compiled([])[0].device.type == "cuda"
 
+        @config.patch({"triton.cudagraphs": True})
+        def test_no_device_idx_repro_cudagraphs(self):
```

Comment:
this doesn't seem to test copy op that you changed

Reply:
cudagraphs fail for me on multiple devices. I have that skipped in new implementation, but not the existing one. When I do run the existing impl with multiple devices I get 

> RuntimeError: Triton Error [CUDA]: invalid resource handle

Do you get that too?